vignettes/model.Rmd

```diff
@@ -88,68 +88,93 @@ We model the expected number of infections/latent notifications ($\lambda^l$) us
+
+  - **Parametric baseline hazard $\gamma_{g,t,d}$**: hazard derived from a parametric delay distribution, with parameters depending on covariates at the time of reference
+  - **Non-parametric reference time effect $\delta_{g,t,d}$**: effect on the hazard that depends on covariates at the time of reference
+  - **Non-parametric report time effect $\epsilon_{g,t,d}$**: effect on the hazard that depends on covariates at the time of report
+
+Each component adds to the overall hazard through a logistic regression
+\begin{equation}
+  \text{logit} (h_{g,t,d}) = \gamma_{g,t,d} + \delta_{g,t,d} + \epsilon_{g,t,d},
+\end{equation}
+
+where the maximum delay hazard is $h_{g,t,D}=1$, in order to enforce the assumption that all observations are reported within the specified maximum delay. In the following, we describe the parameterisation of the different components.
+
+#### Parametric baseline hazard
+The parametric baseline hazard $\gamma_{g,t,d}$ for a case in group $g$ with reference time $t$ is chosen according to a certain parametric probability distribution with parameters $\mu_{g,t}$ and $\upsilon_{g,t}$. By default, this is a discretised log-normal distribution, but other distributions are supported. To discretise the continuous probability distribution into discrete probabilities $p^{\prime}_{g,t,d}$, we use its cumulative distribution function $F^{\mu_{g,t}, \upsilon_{g,t}}$ to compute
 
-The first component ($\gamma_{g,t,d}$) we estimate what would be the probability of reporting $p^{\prime}_{g,t,d}$ at a given time if it followed a parametric distribution, here implemented using a discretised log normal (a range of other distributions are supported by the package) with the log mean and log standard deviation defined using an intercept and arbitrary shared, reference time indexed, covariates with fixed ($\beta_{f,i}$) and random ($\beta_{r,i}$) coefficients (note these can include auto-regressive terms),
+$$p^{\prime}_{g,t,d} = \frac{F^{\mu_{g,t}, \upsilon_{g,t}}(d+1) - F^{\mu_{g,t}, \upsilon_{g,t}}(d)}{F^{\mu_{g,t}, \upsilon_{g,t}}(D)}.$$
+In words, the discrete probability mass function (pmf) is obtained from the continuous distribution by dividing it into $D$ different bins $$(0,1),\, (1,2),\, \ldots,\, (D-1,D),$$
+where $D$ is the maximum delay covered by the model (specified in `epinowcast` via the `max_delay` argument). Importantly, the bins are normalized by $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$, which ensures that the $p^{\prime}_{g,t,d}$ sum to 1. Since $F^{\mu_{g,t}, \upsilon_{g,t}}(D)$ is the probability of reporting before the maximum delay, this can also be interpreted as conditioning our distribution on the maximum delay. Note that because of the discretisation and normalization, the discrete random variable we obtain generally does not have exactly the same moments as the original, continuous distribution.
```

Comment:
I tried to make the description of the discretisation process more concise and at the same time the whole notation more accurate, as originally we had a statement like `p'_{g,t,d} ~ LogNormal(\mu_{g,t}, \upsilon_{g,t})`, which is not quite right

Reply:
I really like the new structure